setup-redis-cluster.sh

```diff
@@ -82,7 +82,7 @@ for port in "${PORTS[@]}"
 
             sleep 1
 
-            ruskit destroy localhost:${port}
+            redis-clu reset localhost:${port}
```

Comment:
reset <node> will be flushing cluster without destroying.
using --hard 1 destroys cluster.

`redis-clu reset localhost:${port} --hard 1` @zekzekus